### ISSUE TITLE
Chart archive has to be put into main directory for the test discovery to work

### ISSUE THREAD

**Issue by piontec** [giantswarm/roadmap#2108]
We should discover the tests relative to the executing directory, but allow to upload the chart from whereever it is.

**Comment by weatherhog** [comment]
@piontec is this issue still valid?

**Comment by piontec** [comment]
Yes, it's valid, but this brings back a question about ats' usefulness and future in general.

**Comment by weatherhog** [comment]
@piontec shall we discuss the usefulness and future of ATS? Or is this already dead?

**Comment by piontec** [comment]
It's in limbo: we decided we don't want to work on it anymore, but we have no alternative that supports local (kind based) devloper flows
